mort_cal.py

```diff
@@ -0,0 +1,38 @@
+#fixed variables
+## Fixed variables are mostly unliked, as we don't have good reasons typically to make these
+## fixed vars. Usually, you want this passed in by a user or you want the application to read
+## the variables from a config file, where they can be modified instead
+interest_rate = 3.75
+property_tax = 0.0167
+numberPaymentsPerYear = 12
+term = 30
+totalMonths = numberPaymentsPerYear * term
+escrow = int(1046.88-257.36)
+principalPercent = .2
+affordableMortgage = 1500
+inputMessage = "Input New Home Price"
+successMessage = "Yay! You can afford this house!"
+failureMessage = "Bummer. Try again."
+
+def mortgage(principal):
+    rate_adjust = interest_rate/1200
+    rate_monthly = (1+rate_adjust)**totalMonths
+    prin_form = (rate_adjust*rate_monthly)/(rate_monthly-1)
```

Comment:
You don't want to take and make words shorter for the variables. It seems like a good idea, but eventually, someone will come around that won't understand what the variable is actually trying to say. This is especially in true in tech when working with distributed teams and people who are not native english speakers. All code should tell a story, and we want to make the story as concise as possible.